server/src/resolvers/usersResolver.ts

```diff
@@ -0,0 +1,102 @@
+import {UserRepository} from "../repositories/Users/UserRepository";
+
+//TODO: Update all "args" parameters upon implementation
+const UsersResolvers = {
+    Query: {
+      user: async (_: any, args: any, context: any) => {
+        try {
+          const ur = new UserRepository();
+          return await ur.getUserByID(args.id);
+          //multiple users?
+        } catch (e) {
+          console.log("Error:", e);
+        }
+      },
+    },
+
+    Mutation: {
+        addStudentUser: async (_: any, args: any) => {
+          try {
+            const ur = new UserRepository();
+            return await ur.addUser(args)
+          } catch (e) {
+            console.log("Error", e)
+          }
+        },
+        addFacultyUser: async (_: any, args: any) => {
+          try {
+            const ur = new UserRepository();
+            return await ur.addUser(args)
+          } catch (e) {
+            console.log("Error", e)
+          }
+        },
+
+        updateFacultyUser: async (_: any, args: any) => {
+          try {
+            const ur = new UserRepository();
+            await ur.updateUser(args)
+          } catch (e) {
+            console.log("Error", e)
+          }
+        },
+
+        updateStudentUser: async (_: any, args: any) => {
+          try {
+            const ur = new UserRepository();
+            await ur.updateUser(args)
+          } catch (e) {
+            console.log("Error", e)
+          }
+        },
+    
+        addTraining: async (_: any, args: any) => {
```

Comment:
This would just be for admins' use, right? In normal circumstances, once you pass the training module, the backend should automatically mark it as complete for you, without an additional call from the frontend.

Reply:
@MatthewGalan just for my understanding, you envision that a user will take the training quiz and at somepoint their answers are sent to the backend from the client. Then the backend checks the user's answers and if they passed it it'll "addTraining" to that user?

Reply:
Yes I envisioned this for system/admin use only.